### ISSUE TITLE
cdmVersion should allow integers

### ISSUE THREAD

**Issue by alondhe** [OHDSI/Achilles#312]
### Expected behavior
cdmVersion <- 5 shouldn't throw a strsplit error

### Actual behavior
It throws a strsplit error

### Steps to reproduce behavior
Set cdmVersion <- 5, then pass in as a parameter.

**Comment by chrisknoll** [comment]
Actually, I think in this case, we should be using semVer formatting of this value: X.Y.Z.  And 1.2.3 < 1.10.1, so these numbers don't follow integer comparison rules.

I believe that's why we expect strings there.

**Comment by alondhe** [comment]
Agreed, but I'm thinking either we provide a more graceful error ("Use a string cdmVersion") or just check the type and if integer, cast to a string.
